hardware/Wireless-Products-Protocols-and-Applications-CN.md

```diff
@@ -0,0 +1,64 @@
+[English](Wireless-Products-Protocols-and-Applications) | 中文
+
+<details>
+<summary><font size=5>目录</font> </summary>
+
+- [1. 私有协议产品与应用](#1-私有协议产品与应用)
+- [2. Zigbee/Thread协议产品与应用](#2-Zigbee/Thread协议产品与应用)
```

Comment:
The link to Chap2 is not working. (Maybe it's caused by the special character "/")

Reply:
OK. Changed "Zigbee/Thread" to "Zigbee-Thread". Thanks.